File: src/main/java/dev/obscuria/tooltips/client/component/BlankComponent.java
package dev.obscuria.tooltips.client.component;

import net.minecraft.client.gui.FontRenderer;

public final class BlankComponent implements TooltipComponent {

    public static final BlankComponent INSTANCE = new BlankComponent();

    @Override
    public int getWidth(FontRenderer font) {
        return 0;
    }

    @Override
    public int getHeight() {
        return 0;
    }
}

File: common/src/main/kotlin/dev/obscuria/tooltips/client/renderer/BlankComponent.kt
package dev.obscuria.tooltips.client.renderer

import net.minecraft.client.gui.Font
import net.minecraft.client.gui.screens.inventory.tooltip.ClientTooltipComponent

object BlankComponent : ClientTooltipComponent {

    override fun getHeight(): Int = 0

    override fun getWidth(p0: Font): Int = 0
}